lxd/storage/drivers/driver_btrfs_utils.go

```diff
@@ -580,3 +580,8 @@ func (d *btrfs) receiveSubVolume(r io.Reader, receivePath string) (string, error
 
 	return subVolPath, nil
 }
+
+func (d *btrfs) resizeFilesystem(sizeBytes int64) error {
+	_, err := shared.RunCommand("btrfs", "filesystem", "resize", fmt.Sprintf("%d", sizeBytes), GetPoolMountPath(d.name))
```

Comment:
Given that this is one line, I think I'd be tempted to move it into the Update function above.